Resources/Audio/Magic/licenses.txt

```diff
@@ -2,7 +2,13 @@ https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/sound/magic
 https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/sound/magic/blink.ogg
 https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/sound/magic/Knock.ogg
 
+copyright: CC BY-SA 3.0
+
 fireball.ogg taken from /tg/station at https://github.com/tgstation/tgstation/commit/906fb0682bab6a0975b45036001c54f021f58ae7
 disintegrate.ogg from /tg/station at https://github.com/tgstation/tgstation/commit/6fde7a49fcd56d6edb844cd31a04ce026065de8b
 
 copyright: CC BY-SA 3.0
+
+rumble.ogg from Uzbazur at https://github.com/tgstation/tgstation/commit/6fde7a49fcd56d6edb844cd31a04ce026065de8b
+
+copyright: CC BY 4.00
```

Comment:
Please don't add to tech debt with lists, use attributions.yml

Also are you sure that license is even correct for tg, the commit is definitely incorrect.